validation-test/Sema/type_checker_perf/fast/issue-43715.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %target-typecheck-verify-swift -solver-scope-threshold=300
+
+func slow(someOptionalString: String?) {
+  print("\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")\(someOptionalString ?? "")")
```

Comment:
I vaguely remember one with a bunch of ?? operators like that in the suite, could you please double-check if we already have it?